README.md

```diff
@@ -11,8 +11,8 @@
 Chef cookbook that provides resources for managing HPE OneView.
 
 ## Requirements
- - Chef 12.0 or higher
- - For oneview resources: HPE OneView 2.0 or 3.0 (API versions 200 or 300). May work with other versions too, but no guarantees
+ - Chef 13.0 or higher
```

Comment:
Is this purposely different from `gem 'chef', '>= 12.0'`?

Reply:
Nope, I will change it.